internal/pkg/cli/env_init.go

```diff
@@ -329,12 +329,11 @@ func (o *initEnvOpts) validateCustomizedResources() error {
 		return fmt.Errorf("cannot import or configure vpc if --%s is set", defaultConfigFlag)
 	}
 	if o.importVPC.isSet() {
-		// We allow 0 or 2+ public subnets.
+		// Allow passing in VPC without subnets, but error out early for too few subnets-- we won't prompt the user to select more of one type if they pass in any.
 		if len(o.importVPC.PublicSubnetIDs) == 1 {
-			return fmt.Errorf("at least two public subnets must be imported to enable Load Balancing")
+			log.Warningf("At least two public subnets must be imported to enable Load Balancing.\n")
```

Comment:
I wonder if we should return an error here.

If we error out when we can only find one public subnet: https://github.com/aws/copilot-cli/blob/ab91cd60a0e526cb9198e941b2f3f42c3a9df923/internal/pkg/cli/env_init.go#L486 
```
if len(publicSubnets) == 1 {
   return errors.New("select public subnets: at least two public subnets must be selected to enable Load Balancing")
}
```
during `Ask()`, then we should probably also error out here as well.

Reply:
Hm. I wonder if instead we should not error out in `Ask()` (see my response to Efe, above)

Reply:
+1 sorry I just saw this chain of comments haha, I think it makes sense to err if they only choose 1 public subnet. It's most likely going to lead to problems rather than make them happy. If they want any public subnets (at least 2 most be selected), otherwise they can completely opt out.